### ISSUE TITLE
LinkedIn lookalike hashtagging

### ISSUE THREAD

**Issue by kalemi19** [signavio/react-mentions#396]
I'm trying to replicate how hashtag suggestions appear on LinkedIn using react-mentions, but can't seem to get it to work, nor sure if it's even possible.

<img width="567" alt="Screen Shot 2020-05-05 at 10 47 16 PM" src="https://user-images.githubusercontent.com/1066392/81114668-c1b8df80-8f22-11ea-9f6c-22c87e51fc76.png">

Basically LinkedIn highlights the hashtag as you type, regardless if you choose one from the suggestions or not. You can also hit the backspace button without deleting the whole hashtag.

This is my setup based on some threads I read here.

   ```
   <Mention
        trigger="#"
        markup="#__id__"
        regex={/#(\S+)/}
        data={[{
          id: 'covid19',
          display: 'covid19',
        },{
          id: 'covidsafe',
          display: 'covidsafe',
        }]}
        appendSpaceOnAdd
        style={{ color: colorPalette.secondary, zIndex: 1, position: 'relative' }}
        displayTransform={(id, display) => `#${display}`}
        renderSuggestion={suggestion => Mentions.renderHashtagsSuggestion(suggestion)}
      />
```

<img width="824" alt="Screen Shot 2020-05-06 at 12 11 38 AM" src="https://user-images.githubusercontent.com/1066392/81121045-2f1e3d80-8f2e-11ea-89ac-e68d980075e6.png">

Unfortunately the suggestions go away as soon as I type #c, it gets highlighted and if I press backspace, the whole thing gets deleted.

<img width="839" alt="Screen Shot 2020-05-06 at 12 11 11 AM" src="https://user-images.githubusercontent.com/1066392/81121055-347b8800-8f2e-11ea-9a25-51e65f3ec4fa.png">

I'm using v3.0.2

**Comment by alDoner** [comment]
> You can also hit the backspace button without deleting the whole hashtag.

I agree with this. Is there any way to achieve this kind of behavior?
